core/app/[locale]/(default)/product/[slug]/page-data.ts

```diff
@@ -171,6 +171,7 @@ const ProductQuery = graphql(
     query ProductQuery($entityId: Int!) {
       site {
         product(entityId: $entityId) {
+          sku
```

Comment:
So the sku varies by product options selected. This query specifically does not receive the product options as variables so it won't fetch the SKU for a specific variant. I guess this would return the SKU of the parent product. Is this enough for your needs?

Reply:
removed sku